spec/src/main/asciidoc/core/spi.asciidoc

```diff
@@ -131,13 +131,28 @@ public interface ObserverMethod<T> {
 * `getObservedType()` and `getObservedQualifiers()` return the observed event type and qualifiers.
 * `getReception()` returns `IF_EXISTS` for a conditional observer and `ALWAYS` otherwise.
 * `getTransactionPhase()` returns the appropriate transaction phase for a transactional observer method or `IN_PROGRESS` otherwise.
-* `getPriority()` returns the priority that will be used by the container to determine the notification order.
+* `getPriority()` this method inherited from <<prioritized, `Prioritized` interface>> returns the priority that will be used by the container to determine the notification order.
```

Comment:
What exactly do you mean? If users choose to implement this interface, they will have to add _some_ value. They needn't really care for default. Or am I missing something?